Ubuntu & macOS builds failing

It looks like all recent [Ubuntu builds](https://github.com/3rdparty/eventuals-grpc/actions/workflows/ubuntu.yml): and [macOS builds](https://github.com/3rdparty/eventuals-grpc/actions/workflows/macos.yml) have been failing for a few weeks.

Sample error: https://github.com/3rdparty/eventuals-grpc/runs/4608427962?check_suite_focus=true

Run bazel build --spawn_strategy=local --copt="-g" --strip="never" :grpc
  bazel build --spawn_strategy=local --copt="-g" --strip="never" :grpc
  shell: /usr/bin/bash -e {0}
  env:
    LLVM_PATH: /home/runner/work/eventuals-grpc/eventuals-grpc/llvm
    LD_LIBRARY_PATH: /home/runner/work/eventuals-grpc/eventuals-grpc/llvm/lib:
    DYLD_LIBRARY_PATH: /home/runner/work/eventuals-grpc/eventuals-grpc/llvm/lib:
Extracting Bazel installation...
Starting local Bazel server and connecting to it...
Loading: 
Loading: 0 packages loaded
ERROR: Error fetching repository: java.io.IOException: No WORKSPACE file found in /home/runner/.cache/bazel/_bazel_runner/a7f112ba8f19a25df8d20f939434879e/external/com_github_3rdparty_eventuals
ERROR: no such package '@com_github_3rdparty_eventuals//bazel': No WORKSPACE file found in /home/runner/.cache/bazel/_bazel_runner/a7f112ba8f19a25df8d20f939434879e/external/com_github_3rdparty_eventuals
INFO: Elapsed time: 4.158s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (0 packages loaded)
FAILED: Build did NOT complete successfully (0 packages loaded)
Error: Process completed with exit code 1.

--- a/.github/workflows/macos.yml
+++ b/.github/workflows/macos.yml
@@ -20,6 +20,8 @@ jobs:
     steps:
       # Checkout the repository under $GITHUB_WORKSPACE.
       - uses: actions/checkout@v1
+        with:
+          submodules: true
 
       - name: Check for bazel
         run: bazel --version

--- a/.github/workflows/ubuntu.yml
+++ b/.github/workflows/ubuntu.yml
@@ -20,6 +20,8 @@ jobs:
     steps:
       # Checkout the repository under $GITHUB_WORKSPACE.
       - uses: actions/checkout@v1
+        with:
+          submodules: true
 
       - name: Check for bazel
         run: bazel --version

--- a/.github/workflows/windows.yml
+++ b/.github/workflows/windows.yml
@@ -20,6 +20,8 @@ jobs:
     steps:
       # Checkout the repository under $GITHUB_WORKSPACE.
       - uses: actions/checkout@v1
+        with:
+          submodules: true
 
       - name: Check for bazel
         run: bazel --version

